scanpipe/pipes/purldb.py

```diff
@@ -320,3 +325,74 @@ def populate_purldb_with_discovered_dependencies(project, logger=logger.info):
         chunk_size=10,
         logger=logger,
     )
+
+
+def send_project_json_to_matchcode(
+    project_json_location, timeout=DEFAULT_TIMEOUT, api_url=PURLDB_API_URL
+):
+    """
+    Given a path to a ScanCode.io Project JSON output, `project_json_location`,
+    send the contents of the JSON file to PurlDB for matching.
+    """
+    project_json_contents = open(project_json_location, "rb")
+    files = {"upload_file": project_json_contents}
+
+    response = request_post(
+        url=f"{api_url}matching/",
+        data={},
+        timeout=timeout,
+        files=files,
+    )
+
+    return response
+
+
+def match_to_purldb(project):
+    """
+    Given a `project` where `scan_output_location` has been set, send the scan
```

Comment:
What if `project.scan_output_location` was not set? We may want to make that step part of this function.

Reply:
I removed `project.scan_output_location` after adding a line to get scan results from the project to the `send_project_json_to_matchcode` step.